app/stac_api/serializers.py

```diff
@@ -468,12 +468,10 @@ class Meta:
         required=False, max_length=255, allow_null=True, allow_blank=False
     )
     description = serializers.CharField(required=False, allow_blank=False, allow_null=True)
-    type = serializers.ChoiceField(
-        source='media_type',
-        required=True,
-        choices=MEDIA_TYPES_MIMES,
-        allow_null=False,
-        allow_blank=False
+    # Can't be a ChoiceField, as  the validate method normalizes the MIME string only after it
+    # is read. Consistency is nevertheless guaranteed by the validate method.
+    type = serializers.CharField(
+        source='media_type', required=True, allow_null=False, allow_blank=False
```

Comment:
Does this have an impact on the admin interface ? In the admin interface we had a dropdown button, for the admin user this should be kept as dropdown button.